src/cli.js

```diff
@@ -315,10 +315,18 @@ export default async function run(tyr) {
   } catch (err) {
     log.error('Failed to generate your project!', err);
   } finally {
-    await deleteGitHubToken(
-      configs.credentials.github.url,
-      configs.credentials.github.username,
-      configs.credentials.github.password
-    );
+    if (typeof configs.credentials.github !== 'undefined') {
+      try {
+        await
```

Comment:
Why are we awaiting this? Wouldn't it be better to make it async and then just call .catch on the promise?

Reply:
He said that it might hang so we want to await it since it is the last thing we will have to do in the program before it exits